### PR TITLE
Microphysics emulation piggy backed diagnostics.

### LINKED ISSUE #1454: Save piggy backed diagnostics in [tile,y,x] format.

### PR DESCRIPTION
**Pull request by nbren12**
Resolves #1454

### CODE REVIEW

File: projects/microphysics/scripts/piggy-back.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ds = vcm.fv3.metadata.gfdl_to_standard(piggy).merge(grid)
+ds["time"] = np.vectorize(vcm.cast_to_datetime)(ds.time)
+
+metrics = {}
+
+for field in ["cloud_water", "specific_humidity", "air_temperature"]:
+    emulator_var = f"tendency_of_{field}_due_to_zhao_carr_emulator"
+    physics_var = f"tendency_of_{field}_due_to_zhao_carr_physics"
+
+    fv3viz.plot_cube(ds.isel(time=0, z=50), emulator_var)
```
Comment (frodre) [inline projects/microphysics/scripts/piggy-back.py:49]: Might be useful in the future to be able to run this for more than one level or time.  Not necessary for current PR though.

File: projects/microphysics/scripts/prognostic_run.py [hunk 2]
```diff
@@ -41,7 +28,6 @@ def get_env(args):
 CONFIG_PATH = Path(__file__).parent / "fv3config.yml"
 
 parser = argparse.ArgumentParser()
-parser.add_argument("--duration", type=str, default="1d")
```
Comment (frodre) [inline projects/microphysics/scripts/prognostic_run.py:44]: Where'd duration go?

Comment (frodre) [reply projects/microphysics/scripts/prognostic_run.py:44]: NVM, see it in the config now

**Review by frodre — APPROVED** [review]
I didn't test it out, but looks fine to me.

**Comment by nbren12** [comment]
Here is an example of the piggy backed diagnostics: https://wandb.ai/ai2cm/microphysics-emulation/runs/2hxkx27m?workspace=user-nbren12

I also reworked the inputs to use the "tag" as an the input that can be used to run the jobs in sequence.

**Review by frodre — APPROVED** [review]
Looks good to me.  I'm guessing the emulation restart/diagnostic regression failure is from the ZC restart fix?  If so, just make sure to update the regtest files there.

Thanks!

**Comment by nbren12** [comment]
I added the surface precipitation diagnostics upstream: https://github.com/ai2cm/fv3gfs-fortran/pull/244

I might wait to merge this PR until that merges (hopefully tomorrow AM).

**Pull request merged**